packages/shared/locales/en.json

```diff
@@ -1950,6 +1950,9 @@
             "description": "Optional parameter: A description of the NFT",
             "attributes": "Optional parameter: An array of traits and values that define attributes of the NFT",
             "uri": "To create a URI using custom media, first upload your file to IPFS via a storage service (e.g. https://nft.storage/)"
+        },
+        "governance": {
+            "removeProposalWarning": "You must stop voting for this proposal in all accounts before removing it."
```

Comment:
Not a requested change, but this makes me think we should move the proposal registration to be account-based rather than profile-based ASAP.